src/types.js

```diff
@@ -120,12 +152,12 @@ export type AtmosphereMessage = {
 };
 
 export type WebsocketCallbacks = {
-  onConnectionLoss?: () => any,
-  onConnectionEstablish?: () => any,
-  onMessage?: (message: AtmosphereMessage) => any,
-  onTransportFailure?: (errorMsg: string, req: AtmosphereRequest) => any,
-  onClose?: () => any,
-  onBurn?: (burnData: BurnItDownResponse) => any,
+  onConnectionLoss?: () => void,
+  onConnectionEstablish?: () => void,
+  onMessage?: (message: AtmosphereMessage) => void,
+  onTransportFailure?: (errorMsg: string, req: AtmosphereRequest) => void,
+  onClose?: () => void,
+  onBurn?: (burnData?: BurnItDownResponse) => void,
```

Comment:
Though I actually removed it everywhere, so we never pass the data along because it doesn't really offer up much.  Will update flowtype to reflect this